packages/plugins/experimental/plugin-script/src/templates/gmail.ts

```diff
@@ -11,40 +11,42 @@ import {
   // @ts-ignore
 } from 'https://esm.sh/@effect/platform@0.77.2?deps=effect@3.13.3';
 // @ts-ignore
+import { format, subDays } from 'https://esm.sh/date-fns@3.3.1';
+// @ts-ignore
 import { pipe, Chunk, Effect, Ref, Schedule, Stream } from 'https://esm.sh/effect@3.13.3';
 
-// TODO(ZaymonFC): Calculate this dynamically and expose a parameter.
-const DEFAULT_AFTER = '2025-01-01';
-
 export default defineFunction({
   inputSchema: S.Struct({
+    mailboxId: S.String,
+    userId: S.optional(S.String).pipe(S.withDecodingDefault(() => 'me')),
```

Comment:
do defaults like this work for echo objects too?

Reply:
still not yet